src/components/home/HomePage.js

```diff
@@ -1,41 +1,71 @@
-import React, { Component } from 'react';
+import React, { useState, useEffect } from 'react';
 import './HomePage.css';
 
-class HomePage extends Component {
-  render() {
-    return (
-      <div className="HomePage">
-        <div className="uk-grid">
-          <div className="uk-width-1-5@s" style={{ background: '#0e0b0bb3', color: 'white' }}>
-            <div className="uk-offcanvas-content">
-              <button
-                className="uk-button uk-button-default uk-margin-small-right"
-                type="button"
-                data-uk-toggle="target: #offcanvas-push"
-              >
-                Push
-              </button>
+function HomePage() {
+  const [data, setData] = useState(null);
```

Comment:
We need a separate issue to make implement https://aviaryan.com/blog/react-stale-while-revalidate
@AmanRaj1608

Reply:
@AmanRaj1608 any update here?